doc/conf.py

```diff
@@ -442,7 +442,7 @@ def js_tag_with_cache_busting(js):
     "collapse_navigation": not is_release_build,
     "show_prev_next": False,
     "switcher": {
-        "json_url": "https://matplotlib.org/devdocs/_static/switcher.json",
+        "json_url": f"https://matplotlib.org/devdocs/_static/switcher.json?{SHA}",
```

Comment:
I'll admit I'm a little unsure what this is doing, but I trust you.  OTOH, can you add a comment?

Reply:
Added a note.